test/integration/output_test.py

```diff
@@ -252,3 +252,51 @@ def test_mcoe_filled(fast_out_filled, df_name, expected_nuke_fraction, tolerance
         fast_out_filled.__getattribute__(df_name)()
     )
     assert abs(actual_nuke_fraction - expected_nuke_fraction) <= tolerance
+
+
+def test_pudltabl_pickle(fast_out_annual):
+    """Test that PudlTabl is serializable with pickle.
+
+    Because pickle is insecure, bandit must be quieted for this test.
+    """
+    import pickle  # nosec
+    from io import BytesIO
+
+    plants = fast_out_annual.plants_eia860()
+    # just to make sure we keep all the parts
+    keys = set(fast_out_annual.__dict__.keys())
+
+    # dump the object into a pickle stored in a buffer
+    pickle.dump(fast_out_annual, buffer := BytesIO())
+    # restore the object from the pickle in the buffer
+    restored = pickle.loads(buffer.getvalue())  # nosec
+
+    pd.testing.assert_frame_equal(restored._dfs["plants_eia860"], plants)
+    assert set(restored.__dict__.keys()) == keys
+
+
+def test_pudltabl_datazip(fast_out_annual):
+    """Test that PudlTabl is serializable with DataZip.
+
+    If etoolbox is installed, test that PudlTabl can be serialized with DataZip.
+    """
+    dz = pytest.importorskip("etoolbox.datazip")
```

Comment:
`etoolbox` isn't in PUDL requirements, so this will always be a skip in our tests. I'm kind of inclined to say we should keep this downstream, so we don't have a confusing "always skip because this doesn't actually apply to our code as is" test in our codebase.

Reply:
Totally fair, I'll remove the test.